gcloud/bigquery/job.py

```diff
@@ -363,7 +483,7 @@ def allow_quoted_newlines(self):
 
     @property
     def create_disposition(self):
-        """Handling for missing destination table.
+        """Define how the back-end handles a missing destination table.
```

Comment:
I think a second sentence within the docstring listing the values is still needed. It wasn't really clear to me how the backend might handle it, but the values make it obvious.